docs/18-guides/02-security/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: App Security Guide
+description: Secure your Nordcraft app by avoiding frontend secrets and using a backend to handle sensitive API operations safely
+---
+
+# How to Keep Your Nordcraft App Secure
```

Comment:
Please can you reformat all headings to use sentence case, e.g. "How to keep your Nordcraft app secure"